quickwit/rest-api-tests/scenarii/es_compatibility/0020-stats.yaml

```diff
@@ -0,0 +1,79 @@
+method: [GET]
+engines:
+  - quickwit
+  - elasticsearch
+endpoint: "gharchive/_stats"
+expected:
```

Comment:
Can we add `size_in_bytes` in the expected values?

Reply:
I'm not sure why, but the value is not stable

Reply:
This is really strange, I would open an issue to track this unexpected behavior :/